stderr of test 'remote-table-ranges.Bug-7089` in directory 'sql/test/BugTracker-2021` itself:


# 10:13:57 >  
# 10:13:57 >  "/usr/bin/python3.9" "remote-table-ranges.Bug-7085.py" "remote-table-ranges.Bug-7085"
# 10:13:57 >  

# builtin opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Oct2020/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	mapi_port = 50000
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# builtin opt 	raw_strings = false
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_listenaddr = all
# cmdline opt 	mapi_port = 33703
# cmdline opt 	mapi_usock = /var/tmp/mtest-86937/.s.monetdb.33703
# cmdline opt 	gdk_dbpath = /tmp/tmpf489g9j5/node1/node1
# builtin opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Oct2020/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	mapi_port = 50000
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# builtin opt 	raw_strings = false
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_listenaddr = all
# cmdline opt 	mapi_port = 44007
# cmdline opt 	mapi_usock = /var/tmp/mtest-86937/.s.monetdb.44007
# cmdline opt 	gdk_dbpath = /tmp/tmpf489g9j5/node2/node2

# 10:13:59 >  
# 10:13:59 >  "Done."
# 10:13:59 >  

